### ISSUE TITLE
🚀 Add testing coverage

### ISSUE THREAD

**Issue by mattwr18** [Human-Connection/Human-Connection#469]
## :rocket: Feature

We should have an idea of how much of our app is being tested, and enforce some minimum testing coverage

### Is your feature request related to a problem? Please describe.

We are merging in too many components/features with inadequate or no testing

### Describe the prefered solution and alternatives you've considered

Coveralls and Travis CI combo

**Comment by mattwr18** [comment]
@roschaefer I have started working on this.
First, I tried to enable `Coveralls`, but it wasn't picking up this repo even with being added as an owner, maybe cause it's private?
Anyways, I was able to get it enabled with `Codecov`, but then I ran into some issues to do with having multiple directories, with multiple package.json files. 
I would like to be able to have one command from our root package.json, which will run all the jest tests and then report the coverage.
As I told you, I went to a Meetup last week where they were talking about using monorepos where they talked not only about `yarn workspaces`, but also about `lerna`. It seems like this might solve some of our problems?
https://github.com/lerna/lerna
https://github.com/lerna/lerna/blob/master/FAQ.md
